File: src/main/java/Main/ItemClasses/Weapons/ShortRangeWeapon.java
package Main.ItemClasses.Weapons;

public class ShortRangeWeapon extends Weapon{
    public ShortRangeWeapon(String name, String description, int weight, int valueInGold, int amount, String damage) {
        super(name, description, weight, valueInGold, amount, damage);
        this.iconName = "standardClassIcons/Longsword.png";
    }

    public ShortRangeWeapon() {
        super();
        this.iconName = "standardClassIcons/Longsword.png";
    }
}

File: src/main/kotlin/itemClasses/weapons/ShortRangeWeapon.kt
package itemClasses.weapons

import kotlinx.serialization.Serializable

@Serializable
class ShortRangeWeapon() : Weapon() {
    init {
        standardIconName = "Longsword.png"
    }

    constructor(name: String, description: String, weight: Int, valueInGold: Int, amount: Int, damage: String) : this() {
        this.name = name
        this.description = description
        this.weight = weight
        this.valueInGold = valueInGold
        this.amount = amount
        this.damage = damage
    }
}
